fulp_modules/features/species/beefmen/mob/beefman.dm

```diff
@@ -722,3 +723,34 @@
 /datum/sprite_accessory/beef/mouth/smile2
 	name = "Smile2"
 	icon_state = "smile2"
+
+/obj/item/organ/tongue/beefman_tongue
+	name = "meaty sausage tongue"
+	desc = "A meaty and thick muscle typically found in Beefmen."
+	icon = 'icons/obj/food/food.dmi'
+	icon_state = "raw_sausage"
```

Comment:
☹️

Reply:
maaaaaaaaan where do I find the ICON

Reply:
is it in the beefman_bodyparts.dmi ive scoured that file up and down and theres NOTHING that remotely looks like a tongue!!!!!!!

Reply:
https://github.com/fulpstation/fulpstation/commit/7755abdc0595bdba9747a1f9f44f60dfa2468923

Reply:
are we cool now

Reply:
That works